### DISCUSSION TITLE
Do deepgram support reset of timestamp in streaming websocket connection

### DISCUSSION THREAD

**Discussion by k0p038x** [deepgram/community#899] [General help]
Context: We use deepgram speech to text to transcribe calls in real time. We are trying to reuse websocket connections across calls. the only issue i see re-using is the timestamps. Deepgram is sending start and duration along with each transcript response. Is there any way to reset it, so the same session can be used for next call.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
